Examples/Scripts/GsfDebugger/src/widgets.py

```diff
@@ -0,0 +1,161 @@
+import matplotlib
+
+matplotlib.use("QtAgg")
+
+from PyQt5 import QtCore, QtWidgets, QtGui
```

Comment:
This needs Qt? Do you want to supply a reuirements file for this in the `GsfDebugger` folder?

Reply:
yeah good point, I will provide one